debian/control

```diff
@@ -38,6 +38,21 @@ Depends:
 Description: Updater for Endless OS
  This package contains the components for keeping Endless OS up to date.
 
+Package: eos-updater-tests
+Section: misc
+Architecture: any
+Depends:
+ eos-updater (= ${binary:Version}),
+ gir1.2-glib-2.0,
+ python3-gi,
+ ${misc:Depends},
+ ${python3:Depends},
+ ${shlibs:Depends},
+Description: Updater for Endless OS - tests
+ This package contains the components for keeping Endless OS up to date.
```

Comment:
I see; I did not realize that this is also how the other subpackages work. Let's keep it like this for now.